src/pgduckdb_planner.cpp

```diff
@@ -120,6 +120,14 @@ DuckdbPlanNode(Query *parse, const char *query_string, int cursor_options, Param
 		return nullptr;
 	}
 
+	/*
+	 * If creating a plan for a scrollable cursor, make sure it can run
+	 * backwards on demand.  Add a Material node at the top at need.
```

Comment:
```suggestion
	 * backwards on demand. Add a Material node at the top as needed.
```

Reply:
This comment came straight from the postgres source code. Reworded it, because it actually didn't reflect some of the changes I made to the code.